test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -669,6 +670,48 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			})
 		})
 
+		When("perPodPowerManagent enabled", func() {
+			FIt("should update kernel arguments and tuned accordingly", func() {
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+					PerPodPowerManagement: pointer.BoolPtr(true),
+					HighPowerConsumption:  pointer.BoolPtr(false),
+					RealTime:              pointer.BoolPtr(true),
+				}
+				By("Patching the performance profile with workload hints")
+				workloadHints, err := json.Marshal(profile.Spec.WorkloadHints)
+				Expect(err).ToNot(HaveOccurred())
+
+				Expect(testclient.Client.Patch(context.TODO(), profile,
+					client.RawPatch(
+						types.JSONPatchType,
+						[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec/workloadHints", "value": %s }]`, workloadHints)),
+					),
+				)).ToNot(HaveOccurred())
+
+				By("Applying changes in performance profile and waiting until mcp will start updating")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+
+				By("Waiting when mcp finishes updates")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+
+				By("Verifying node kernel arguments")
+				cmdline, err := nodes.ExecCommandOnMachineConfigDaemon(&workerRTNodes[0], []string{"cat", "/proc/cmdline"})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(cmdline).To(ContainSubstring("intel_pstate=passive"))
```

Comment:
As above, I am not sure about this part.

Reply:
same